ONSdigital/census-rm-toolbox
# Ask for user credentials for PSQL commands

## Context
We want to be accessing the DB with user creds rather than using the service account

## What has Changed
* Add dosql script which asks for user creds
* Update write DB access script to ask for user creds
* Update aliases

## How to Test
Build and push this image with a branch tag, deploy to a project and try the dosql command. It should prompt for a username and password which it should then authenticate with.

## Links
https://trello.com/c/u6uTsczT/1212-census-rm-toolbox-db-access-to-use-personal-accounts-instead-of-service-accounts
